.circleci/config.yml

```diff
@@ -33,8 +33,9 @@ jobs:
                 name: Build the documentation
                 command: |
                     cd doc
+                    make clean
                     make html
-                no_output_timeout: 15m
+                no_output_timeout: 20m
```

Comment:
I don't think it's a good idea to hack this too much. You should ideally have computations which take less time ... otherwise it will get harder and harder to debug things

Reply:
> I don't think it's a good idea to hack this too much. You should ideally have computations which take less time ... otherwise it will get harder and harder to debug things

It was only a proof to see if it was a time problem but it seems this crash is not related to time...I don't have idea why circleci doesn't work properly :(